epictrack-web/src/components/workPlan/status/StatusView/RecentStatus.tsx

```diff
@@ -1,172 +1,169 @@
+            .format(MONTH_DAY_YEAR)
+            .toUpperCase()}
+        </ETCaption1>
+        <If condition={!statuses[0].is_approved}>
+          <Then>
+            <ETCaption1
+              bold
+              sx={{
+                color: Palette.error.dark,
+                backgroundColor: Palette.error.bg.light,
+                padding: "4px 8px",
+                borderRadius: "4px",
+              }}
+            >
+              Need Approval
+            </ETCaption1>
+          </Then>
+          <Else>
+            <When condition={statuses[0].is_approved}>
+              <ETCaption1
+                bold
+                sx={{
+                  color: Palette.success.dark,
+                  backgroundColor: Palette.success.bg.light,
+                  padding: "4px 8px",
+                  borderRadius: "4px",
+                }}
+              >
+                Approved
+              </ETCaption1>
+            </When>
+          </Else>
+        </If>
+      </Box>
+      <ETPreviewText
+        color={Palette.neutral.dark}
+        sx={{ paddingTop: "16px", whiteSpace: "pre-wrap" }}
+      >
+        {statuses[0].description}
+      </ETPreviewText>
+      <Box
+        sx={{
+          display: "flex",
+        }}
+      >
+        <If condition={!statuses[0].is_approved}>
+          <Then>
+            <Button
+              startIcon={<CheckCircleIcon />}
+              onClick={() => {
+                setStatus(statuses[0]);
+                setShowApproveStatusDialog(true);
+              }}
+              sx={{
+                backgroundColor: "inherit",
+                borderColor: "transparent",
+              }}
+            >
+              Approve
+            </Button>
+          </Then>
+          <Else>
+            <Button
+              startIcon={
+                <AddIcon style={{ fill: Palette.primary.accent.main }} />
+              }
+              onClick={() => {
+                setStatus(statuses[0]);
+                setIsCloning(true);
+                setShowStatusForm(true);
+              }}
+              sx={{
+                backgroundColor: "inherit",
+                borderColor: "transparent",
+              }}
+            >
+              New Update
+            </Button>
+          </Else>
+        </If>
+        <If condition={statuses[0].is_approved}>
+          <Restricted
+            allowed={[ROLES.EXTENDED_EDIT]}
+            errorProps={{ disabled: true }}
+          >
+            <Button
+              startIcon={<PencilEditIcon />}
+              onClick={() => {
+                setShowStatusForm(true);
+                setStatus(statuses[0]);
+              }}
+              sx={{
+                backgroundColor: "inherit",
+                borderColor: "transparent",
+              }}
+            >
+              Edit
+            </Button>
+          </Restricted>
+          <Else>
+            <Button
+              startIcon={<PencilEditIcon />}

```

Comment:
the changes were adding this startIcon, the MUI way, instead of including the icons in the children